package.json

```diff
@@ -40,10 +40,7 @@
     "url": "https://github.com/resin-io-modules/etcher-sdk/issues"
   },
   "dependencies": {
-    "@types/bluebird": "^3.5.23",
-    "@types/file-type": "^5.2.1",
     "@types/node": "^6.0.112",
```

Comment:
Shouldn't this be in devDependencies too?
Other than that, lgtm

Reply:
@thundron no, we still need this as we use node types (streams and buffers) in exported functions.
See https://stackoverflow.com/questions/45176661/how-do-i-decide-whether-types-goes-into-dependencies-or-devdependencies